app/views/carts/show.html.erb

```diff
@@ -10,6 +10,7 @@
       <th><h3>Price</h3></th>
       <th><h3>Remove Item</h3></th>
       <th><h3>Quantity</h3></th>
+      <th><h3>Subtotal</h3></th>
```

Comment:
nice. I wonder if we have to have subtotals for each item on our past orders page. 

But we'll get there when we get there